resources/public/api/conf/1.0/docs/testing.md

```diff
@@ -1 +1,8 @@
-TBC
\ No newline at end of file
+You can use the sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing).
+
+It does not currently support [stateful behaviour](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/stateful-behaviour). However, the sandbox service for this API supports successful match responses and error scenarios described in resources.
+
+You can set up organisations test data for this API using the following resources:
+
+* [Create an employer](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/api-platform-test-user/1.0#_create-a-test-user-which-is-an-organisation_post_accordion).
```

Comment:
I think we need to remove these as they look to be individuals employments specific. They also reference the DES stub which we know to be incorrect. 

Perhaps just replace with `TBC` under `You can set up organisations test data for this API using the following resources:` for now?

Reply:
Might want to keep the test user  one but drop the other?